app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id', counter_cache: true
+  has_many :likes, foreign_key: 'post_id'
+  has_many :comments, foreign_key: 'post_id'
+
+  after_create do
+    author.increment!(:posts_count)
+  end
```

Comment:
- You can use the `after_save` Active Callback to call the method that updates the post counter, which should be a private method.

Reply:
oh okay, thanks for the suggestion. Will definitely do it for the next PR.